www/js/submission.js

```diff
@@ -276,9 +276,18 @@ var submission = function($) {
 		},
 
 		updatePreview : function() {
-			$('#sampleLayout p').text(function(index){
+			$('#sampleLayout p').html(function(index){
 				if ($('#description').val().length) {
-					return $('#description').val().substring(0,300);
+					var string = $('#description').val().substring(0,300);
+					console.log(string);
+					//Purify it to match what the actual output would look like
+					string = DOMPurify.sanitize(string, {
+						ALLOWED_TAGS: ['a','strong','p','em'],
```

Comment:
Probably best to hoist these configuration settings to the top of the file where they can be quickly changed.